### ISSUE TITLE
css custom properties does not parse correctly 

### ISSUE THREAD

**Issue by barak007** [postcss/postcss#1705]
Here are few cases that postcss does not parse custom properties correctly the last one has the most potential to create bugs since the value does not contain the space and comments are easily can be removed. 
```css
/* decl.value should not contain comments (raws.value should) (minor bug since value contain the space) */
.root{--x: /*asd*/;}
/* decl.value should not contain comments (I guess it's compatible with the first issue) (minor bug since value contain the space) */
.root{--x:/*asd*/ ;}
/* this comment is parsed as comment node and the value is empty it should contain one space (major bug since the value does not contains the space) */
.root{--x:/*asd*/ }
```

**Comment by ai** [comment]
Fixed.

Parser: https://github.com/postcss/postcss/commit/be42a7c250947165a9d0127cd1a06d8c01550e3f
Tests: https://github.com/postcss/postcss-parser-tests/commit/1af55cf4a35d07b3c31389953a2e37a3b00b0b47

Can you double-check tests changes and GitHub’s version of `postcss` on your codebase?

**Comment by barak007** [comment]
Did the check.
This seems to be much better!. since I was working around this issue I had to change the code but for the good.
There is still a thing that I'm not sure if needed to address.
The browser only keeps one space between each value token of a custom property.
I'm not sure if it's postcss job to normalize them because postcss does not parse the internal value AST.

This is an example (look at the console):
[BIN](https://jsbin.com/xudojod/4/edit?html,css,js,output) 

Bottom line this change feels good to me if you don't thing that the above concern is relevant I'm ok with that

**Comment by ai** [comment]
Released in 8.4.6.
